README.md

```diff
@@ -1,49 +1,47 @@
 # CPAN.org content repository
 
-This repository is for maintaining the non-source-code, non-archive
-content on cpan.org.  Discussion related to this should be on the
-[cpan-workers mailing list][cpan-workers].
+This repository is for maintaining the non-source-code, non-archive content on [cpan.org](https://www.cpan.org).
+Discussion related to this should be on the [**cpan-workers** mailing list](https://lists.perl.org/list/cpan-workers.html).
 
-[cpan-workers]: http://lists.perl.org/list/cpan-workers.html
+## Rules for editing content
 
-## Rules for editing content:
+- Don't add new output files.
+Only files already on cpan.org should be added (and then to their current location).
+Generally email [ask@perl.org](mailto:ask@perl.org) to get new files included here.
+Making new templates/scripts/etc to produce the output files is fine.
 
-* Don't add new output files. Only files already on cpan.org should be
-added (and then to their current location). Generally email
-ask@perl.org to get new files included here. Making new
-templates/scripts/etc to produce the output files is fine.
+- Be conservative in your edits.
```

Comment:
Please don't change formatting arbitrarily.  This can go back to being a * and there's no reason to change the wrapping.  Also above and below.